src/tests/WebApi.MsBuild.SystemTest.ValidateContract/WebApi.MsBuild.SystemTest.ValidateContract.csproj

```diff
@@ -0,0 +1,30 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+    <PropertyGroup>
+        <TargetFramework>net8.0</TargetFramework>
+        <Nullable>enable</Nullable>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <IsPackable>false</IsPackable>
+        <WarnOnPackingNonPackableProject>false</WarnOnPackingNonPackableProject>
+        <RootNamespace>WebApi.MsBuild.SystemTest</RootNamespace>
+    </PropertyGroup>
+
+    <PropertyGroup>
+      <OpenApiDevelopmentMode>GenerateContract</OpenApiDevelopmentMode>
+      <OpenApiCompareCodeAgainstSpecFile>true</OpenApiCompareCodeAgainstSpecFile>
```

Comment:
With the new naming would there be confusion with this property?

Reply:
The behaviour hasn't changed. Do you think this is obsolete? Can it always be true?

Reply:
It could be true by default when you are in the mode ValidateContract, but when you are in the mode GeneratingContract it does compare the spec with the code instead of generating it.

A better experience could be to put some condition:
- Mode: ValidateContract - always true
- Mode: GenerateContract + if in release mode (CI) - set to true